front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -104,44 +100,45 @@ const useOutputTableData = (
         return;
       }
 
-      const formattedPoints = formatSuggestedOperationalPoints(operationalPoints);
-
-      const formattedPointsWithTrackNames = await addTrackNamesToSuggestedOperationalPoints(
-        formattedPoints,
-        getTrackSectionsByIds
-      );
+      const trackIds = operationalPoints.map((op) => op.part.track);
+      const trackSections = await getTrackSectionsByIds(trackIds);
+      const formattedPointsWithTrackNames = operationalPoints.map((op) => ({
+        ...op,
+        trackName: trackSections[op.part.track]?.extensions?.sncf?.track_name,
+      }));
```

Comment:
You don't need to do this map ;)
You can directly get the trackName of the op l114 and return it in the correct object
```suggestion
```

Reply:
Oh right thanks! I was still thinking in terms of the previous function, but there's no longer a need for that ^^